contents/csharp/Xamarin.md

```diff
@@ -0,0 +1,145 @@
+<frontmatter>
+  title: Xamarin
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Xamarin
+
+**Author: [Ni Tianzhen](https://github.com/niqiukun)**
+
+<box id="article-toc">
+
+* [What is Xamarin‎](#what-is-xamarin)
+* [How Xamarin Works?](#how-xamarin-works)
+    * [Xamarin.Android and Xamarin.iOS](#xamarin.android-and-xamarin.ios)
+    * [Xamarin.Forms](#xamarin.forms)
+    * [Xamarin.Essentials](#xamarin.essentials)
+* [Why Xamarin?‎](#why-xamarin)
+    * [Complete Binding for Underlying SDKs](#complete-binding-for-underlying-sdks)
+    * [Objective-C, Java, C, and C++ Interop](#objective-c-java-c-and-c-interop)
+    * [Modern Language Constructs and Robust Base Class Library](#modern-language-constructs-and-robust-base-class-library)
+    * [Mobile Cross-Platform Support](#mobile-cross-platform-support)
+    * [Modern Integrated Development Environment (IDE)](#modern-integrated-development-environment-ide)
+* [How to Get Started‎?](#how-to-get-started)
+</box>
+
```

Comment:
since it is mainly about c#, maybe can put a warning box at top claiming this asuuming readers have sufficient knowledge about c#

Reply:
This article is a brief overview of what Xamarin is and the benefits of it. There isn't much technical details about C#, hence I believe prior knowledge isn't required. If the readers are interested in Xamarin after reading this article, they can follow the last section to read more about C#.